sklearn/model_selection/tests/test_search.py

```diff
@@ -385,20 +384,21 @@ def test_no_refit():
         # error messages
         for fn_name in ('predict', 'predict_proba', 'predict_log_proba',
                         'transform', 'inverse_transform'):
-            assert_raise_message(NotFittedError,
-                                 ('refit=False. %s is available only after '
-                                  'refitting on the best parameters'
-                                  % fn_name), getattr(grid_search, fn_name), X)
+            error_msg = ('refit=False. %s is available only after refitting on'
+                         ' the best parameters' % fn_name)
```

Comment:
Similar comment here: let's use an f-string instead of `%s`-based string interpolation.